### PR TITLE
Can use divine rune pouch in place of rune pouch in inferno

### LINKED ISSUE #4927: Inferno - require rune pouch
## 🐛 Bug Report

Divine rune pouch hasn't been added as a similar item potentially or inferno is specifically looking for the slayer rune pouch

Reported via lewis in testing discord post release

image

### PR DESCRIPTION
**Pull request by Sturminator3**
### Can use divine rune pouch in place of rune pouch in inferno

Closes #4927

### Other checks:

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Comment by LuxrayElite** [comment]
It would probably be best to implement similar code to the tob fix 

https://github.com/oldschoolgg/oldschoolbot/blob/e4b5198ab05a1acee4f5974c97ca5a2481572458/src/mahoji/lib/abstracted_commands/tobCommand.ts#L106-L109

Magna also prefers using for of instead of forEach due to speed :)

**Comment by Sturminator3** [comment]
> 

Sounds good I can make those changes!

**Comment by Sturminator3** [comment]
@LuxrayElite When I add that return statement im getting a bunch of compilation errors in my file. Do you know if there is some sort of setting I need for it?

`return [true, '${user.usernameOrMention}'s doesn't have a Rune pouch.'];`

**Comment by LuxrayElite** [comment]
Saw the comment in gitlogs :P, glad you figured it out.

Much tidier now :) good job on the first PR

**Comment by LuxrayElite** [comment]
Id probably use the old message error message as well just cause the one from ToB pings the user 👍 

image

**Comment by Sturminator3** [comment]
@LuxrayElite Changes have been pushed!

File: src/mahoji/lib/abstracted_commands/infernoCommand.ts [hunk 1]
```diff
@@ -163,13 +161,19 @@ async function infernoRun({
...
 	 * Item Requirements
 	 *
 	 */
-	if (!user.owns(itemRequirements)) {
-		return `To do the Inferno, you need these items: ${itemRequirements}.`;
+	const itemRequirements = getSimilarItems(itemID('Rune pouch'));
+	if (itemRequirements.every(item => !user.owns(item))) {
+		let itemRequirementNames: string[] = [];
+		for (const item of itemRequirements) {
+			itemRequirementNames.push(itemNameFromID(item) as string);
```
Comment (gc) [inline src/mahoji/lib/abstracted_commands/infernoCommand.ts:174]: this typecast shouldnt be needed.

File: src/mahoji/lib/abstracted_commands/infernoCommand.ts [hunk 2]
```diff
@@ -163,13 +161,19 @@ async function infernoRun({
...
+		}
+		return `To do the Inferno, you need one of these items: ${itemRequirementNames.join(', ')}.`;
```
Comment (gc) [inline src/mahoji/lib/abstracted_commands/infernoCommand.ts:176]: this `itemRequirementNames` is a bit odd. Try just `itemRequirements.map(itemNameFromID).join(", ")`

**Review by gc — CHANGES_REQUESTED** [review]

**Comment by Sturminator3** [comment]
@gc PR has been updated with your changes

**Comment by osrsjobbo** [comment]
Ready for review, tests green

**Pull request merged**